app/vep/models/client_model.py

```diff
@@ -1 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+class FieldOptions:
+    GENCODE_LABEL: str = "GENCODE"
+    GENCODE_VALUE: str = "gencode_comprehensive"
+
+class Options(BaseModel):
+  label: str = FieldOptions.GENCODE_LABEL
+  value: str = FieldOptions.GENCODE_VALUE
```

Comment:
Initialise to an empty string rather than to `FieldOptions.GENCODE_LABEL` / `FieldOptions.GENCODE_VALUE`?